README.md

```diff
@@ -11,9 +11,30 @@ npm install -g cdk-import
 
 ## Usage
 
+There are currently two sources that resources can be generated from. The subcommand 
+`cfn` is used to import from CloudFormation Registry, 
+`sc` is used to import AWS Service Catalog products.
+There are shared general options for output directories and target language.
+The `cfn` subcommand is optional as it is default import path.
```

Comment:
This library is still experimental so I think I am fine with a breaking change here to require a subcommand.

Reply:
cool, sounds good, wasn't sure if we could do that